agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -0,0 +1,337 @@
+#!/bin/bash
+
+# Installer for Sysdig Agent on IBM Cloud Kubernetes Service (IKS)
+
+set -e
+
+function install_curl_deb {
+	export DEBIAN_FRONTEND=noninteractive
+
+	if ! hash curl > /dev/null 2>&1; then
+		echo "* Installing curl"
+        $CMD_PREF apt-get update
+		$CMD_PREF apt-get -qq -y install curl < /dev/null
+	fi
+}
+
+function install_curl_rpm {
+	if ! hash curl > /dev/null 2>&1; then
+		echo "* Installing curl"
+		$CMD_PREF yum -q -y install curl
+	fi
+}
+
+function download_yamls {
+    echo "* Downloading Sysdig cluster role yaml"
+	curl -s -o /tmp/sysdig-agent-clusterrole.yaml https://raw.githubusercontent.com/draios/sysdig-cloud-scripts/master/agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml
+	echo "* Downloading Sysdig config map yaml"
+	curl -s -o /tmp/sysdig-agent-configmap.yaml https://raw.githubusercontent.com/draios/sysdig-cloud-scripts/master/agent_deploy/kubernetes/sysdig-agent-configmap.yaml
+	echo "* Downloading Sysdig daemonset v2 yaml"
+	curl -s -o /tmp/sysdig-agent-daemonset-v2.yaml https://raw.githubusercontent.com/draios/sysdig-cloud-scripts/master/agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml
+}
+
+function unsupported {
+	echo "Unsupported operating system. Try using the the manual installation instructions"
+	exit 1	
+}
+
+function help {
+	echo "Usage: $(basename ${0}) -a | --access_key <value> [-t | --tags <value>] [-c | --collector <value>] \ "
+    echo "                [-cp | --collector_port <value>] [-s | --secure <value>] [-cc | --check_certificate] \ "
```

Comment:
Hey, I think this is a problem with indentation, could you please fix it??

Thanks!

Reply:
Fixed!

Reply:
Fixed!